pdk/1.1/modules/ROOT/pages/policies-pdk-upgrade-pdk.adoc

```diff
@@ -0,0 +1,107 @@
+= Upgrading PDK
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+Flex Gateway Policy Development Kit (PDK) is composed of the following separate versioned components that must each be updated to update a custom policy project:
+
+* Anypoint CLI PDK plugin
+* PDK Rust libraries
+* Anypoint Cargo plugin
+
+All your PDK custom policies on your device use the same Anypoint CLI PDK plugin. When you create a new custom policy project, the PDK Rust libraries and Anypoint Cargo plugin versions are the same as the Anypoint CLI PDK plugin used to create the project. The PDK Rust libraries and Anypoint Cargo plugin version of an individual custom policy project are unique from other policy projects. When you upgrade your Anypoint CLI PDK plugin, you must manually upgrade PDK Rust libraries and Anypoint Cargo plugin to be compatible with your Anypoint CLI PDK plugin version.
```

Comment:
I wonder if there's a way to streamline this - or put the important bit first. 

Can you lead with this:

> When you upgrade your Anypoint CLI PDK plugin, you must manually upgrade PDK Rust libraries and Anypoint Cargo plugin to be compatible with your Anypoint CLI PDK plugin version.

And then give the rest as the "reason why"? 

Also, this phrase is duplicated: "the PDK Rust libraries and Anypoint Cargo plugin version"
Any way to combine?

Maybe:
When you create a new custom policy project, the PDK Rust libraries and Anypoint Cargo plugin versions are the same as the Anypoint CLI PDK plugin used to create the project and are unique from other policy projects.